src/Host/Client/Impl/Session/RSessionEvaluationCommands.cs

```diff
@@ -3,6 +3,7 @@
 
 using System;
 using System.Threading.Tasks;
+using Microsoft.R.Host.Client;
 
 namespace Microsoft.R.Host.Client.Session {
     public static class RSessionEvaluationCommands {
```

Comment:
We talked about it, and I'm going to make it so that there is a single interface `IRExpressionEvaluator` that is a single point for this, and then they can be extension methods on that. Currently we have that interface already, but only `RHost` implements it - `IRSession` and `IRSessionEvaluation` do not, even though the functionality is very similar. So we'll unify.